react/components/Charts/LineChart/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react'
+import {
+    Line,
+    LineChart as LineChartBase,
+    XAxis,
+    YAxis,
+    Tooltip,
+    CartesianGrid,
+    ResponsiveContainer,
+} from 'recharts'
+import { colors, defaultProps } from './constants'
+import useChart from '../hooks/useChart'
+
+interface Props {
+    data: any,
```

Comment:
```suggestion
    data: object[],
```
maybe ?

Reply:
Actually, i'm having troubles with proptypes, because i define data with a type in typescript, so when i try to put `object` the proptypes complain of the shape. Do you have some suggestion?